.companion.rc

```diff
@@ -13,10 +13,11 @@ for default_param_file in *; do
 done
 
 if [ ! -f /home/pi/.updating ]; then
-	sudo python $COMPANION_DIR/tools/PingEnumerator.py || true
+	sudo python $COMPANION_DIR/tools/ping_enumerator.py || true
```

Comment:
is ping-enumerator ok?

Reply:
@Williangalvani it's not a module or a package?

Reply:
I usually consider any .py file a module.
I guess technically only when a file is imported it is working as a module, but I'd rather treat it as such.